### PR TITLE
Added primevue components

### PR DESCRIPTION
**Pull request by Rahul139-stack**

### CODE REVIEW

File: application/backend/src/main/resources/apps/typerefinery/components/content/radio/radio.html [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+    See the License for the specific language governing permissions and
+
+    limitations under the License.
+
+*/-->
+
+<!-- <sly data-sly-use.model="io.typerefinery.websight.models.components.Checkbox">
+    <p-checkbox :v-model="${model.showSubtitle}" :binary="true"></p-checkbox>
+    <input type="checkbox"/>
```
Comment (Harikarthyk) [inline application/backend/src/main/resources/apps/typerefinery/components/content/radio/radio.html:33]: remove the commented codes.

File: application/backend/src/main/java/io/typerefinery/websight/models/components/RadioButton.java [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+import org.apache.sling.api.resource.Resource;
+import org.apache.sling.models.annotations.Model;
+import org.apache.sling.models.annotations.injectorspecific.Self;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Arrays;
+import java.util.LinkedHashSet;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
```
Comment (Harikarthyk) [inline application/backend/src/main/java/io/typerefinery/websight/models/components/RadioButton.java:29]: Remove all unused imports.

File: application/backend/src/main/java/io/typerefinery/websight/models/components/Checkbox.java [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+import static org.apache.sling.models.annotations.DefaultInjectionStrategy.OPTIONAL;
+
+import org.apache.sling.api.resource.Resource;
+import org.apache.sling.models.annotations.Model;
+import org.apache.sling.models.annotations.injectorspecific.Self;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Arrays;
+import java.util.LinkedHashSet;
+import java.util.stream.Collectors;
```
Comment (Harikarthyk) [inline application/backend/src/main/java/io/typerefinery/websight/models/components/Checkbox.java:28]: Remove all the unused imports.

File: application/backend/src/main/resources/apps/typerefinery/components/content/radio/radio.html (hunk 1 above)
Comment (Rahul139-stack) [reply application/backend/src/main/resources/apps/typerefinery/components/content/radio/radio.html]: Removed

File: application/backend/src/main/java/io/typerefinery/websight/models/components/RadioButton.java (hunk 2 above)
Comment (Rahul139-stack) [reply application/backend/src/main/java/io/typerefinery/websight/models/components/RadioButton.java]: Removed

File: application/backend/src/main/java/io/typerefinery/websight/models/components/Checkbox.java (hunk 3 above)
Comment (Rahul139-stack) [reply application/backend/src/main/java/io/typerefinery/websight/models/components/Checkbox.java]: Removed

**Pull request merged**